breakout_3_Observability-with-AI/breakout_3_answers/1.1-grafana-assistant.md

```diff
@@ -0,0 +1,7 @@
+## Grafana AI
+Q: How many page loads have there been in the last few hours in my web app?
+
+A: Approximately 1,600-1,780 requests per second over the last 3 hours.
```

Comment:
I am getting significantly more than this - https://badf52.grafana.net/a/grafana-assistant-app/chats/shared/378abfaa-620a-4ad2-8d65-830e6966d8bf

it seems it was using synthetics data from looking at the query. I told it to use frontend o11y and it did better